custom/mlservermllib/models/mllib.py

```diff
@@ -0,0 +1,45 @@
+from mlserver import MLModel, types
+from mlserver.errors import InferenceError
+from .utils import get_mllib_load
+from pyspark import SparkContext, SparkConf
+
+
+class MLlibModel(MLModel):
+
+    async def load(self) -> bool:
+        conf = SparkConf().set('spark.driver.host', '127.0.0.1')
```

Comment:
Do these needs to be configurable? You know way more about Spark than I do, so please feel free to correct me if I'm wrong.

Reply:
The spark conf can be set to local or other spark clusters, ref https://spark.apache.org/docs/latest/submitting-applications.html#master-urls

Since we use mlserver in k8s container, I only add localhost url here right now, but we could make it more  configurable.

Reply:
That makes sense. Let's leave it to localhost for now. We can probably add it to the model parameters if we need to.